cypress/integration/createEventWithoutStage.cy.js

```diff
@@ -0,0 +1,27 @@
+import { DIMENSION_ID_EVENT_DATE } from '../../src/modules/dimensionConstants.js'
+import { AEFI_PROGRAM, TEST_REL_PE_LAST_12_MONTHS } from '../data/index.js'
+import { selectEventProgram } from '../helpers/dimensions.js'
+import { clickMenubarUpdateButton } from '../helpers/menubar.js'
+import { selectRelativePeriod } from '../helpers/period.js'
+import { expectTableToBeVisible } from '../helpers/table.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+const program = AEFI_PROGRAM
+const periodLabel = program[DIMENSION_ID_EVENT_DATE]
+
+describe('event without stage', () => {
```

Comment:
This test is so short and simple I'd move it to the bottom of the `createEvent` file rather than keeping it in a file on its own...

Reply:
If we want multiple `describe`s per file then we can move it.

Reply:
either multiple separate `describe` or we can wrap the two individual ones in an outer `describe` as well (see my suggestion here: https://github.com/dhis2/line-listing-app/pull/217)